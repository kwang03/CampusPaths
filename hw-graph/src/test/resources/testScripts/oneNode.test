#Tests how a graph with one node behaves

#Creates graph
CreateGraph myGraph

#Adds node
AddNode myGraph n1

#Lists children
ListChildren myGraph n1

#Lists nodes
ListNodes myGraph