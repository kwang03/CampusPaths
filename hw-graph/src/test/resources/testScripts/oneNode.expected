#Tests how a graph with one node behaves

#Creates graph
created graph myGraph

#Adds node
added node n1 to myGraph

#Lists children
the children of n1 in myGraph are:

#Lists nodes
myGraph contains: n1